code/__DEFINES/layers.dm

```diff
@@ -110,40 +110,51 @@
 #define LIGHTING_PLANE 15
 #define LIGHTING_LAYER 15
 
-#define RAD_TEXT_LAYER 15.1
+#define LIGHTING_LAMPS_GLARE 16 // Light glare (optional setting)
+#define LIGHTING_EXPOSURE_PLANE 17 // Light sources "cones"
+#define LIGHTING_LAMPS_SELFGLOW 18 // Light sources glow (lamps, doors overlay, etc.)
+#define LIGHTING_LAMPS_PLANE 19 // Light sources themselves (lamps, screens, etc.)
 
-#define ABOVE_LIGHTING_PLANE 16
-#define ABOVE_LIGHTING_LAYER 16
+#define LIGHTING_LAMPS_RENDER_TARGET "*LIGHTING_LAMPS_RENDER_TARGET"
 
-#define FLOOR_OPENSPACE_PLANE 17
-#define OPENSPACE_LAYER 17
+#define RAD_TEXT_LAYER 19.1
```

Comment:
Hey, other reviewers, is there any reason to keep these ugly non-integer hack layers, or should they get flattened out into integers?

Reply:
Ehh, I think it's mostly just so we don't have to keep modifying intermediate layers when adding new ones, since it would probably involve shuffling around a lot of other numbers. I'm in favor of non-integer layers as long as they do indeed land directly between two integers

Reply:
In that case, we could do the same with the ones added here, and avoid the noise of renumbering everything else.